src/hash-table.js

```diff
@@ -7,6 +7,45 @@ class HashTable {
     this.storage = new LimitedArray(this.limit);
     // Do not modify anything inside of the constructor
   }
+  insert(key, value) {
+    const index = getIndexBelowMax(key, this.limit);
+    if (this.storage[index] === undefined) {
+      this.storage[index] = [[key, value]];
+    } else {
+      this.storage[index].push([key, value]);
+    }
+  }
+  retrieve(key) {
+    const index = getIndexBelowMax(key, this.limit);
+    if (this.storage[index] === undefined) {
+      return false;
+    }
+    let valueIn2DArray = null;
```

Comment:
You can just declare `let valueIn2DArray;` to have it be initialized to undefined without having to set it to null, which will achieve the same thing in this case.